src/main/java/info/tehnut/soulshards/item/ItemSoulShard.java

```diff
@@ -39,7 +39,7 @@
 
     static {
         try {
-            Method _getEntityName = MobSpawnerLogic.class.getDeclaredMethod("method_8281");
+            Method _getEntityName = MobSpawnerLogic.class.getDeclaredMethod("getEntityId");
```

Comment:
This will fail in production. It should be migrated to fabrics reflection api. If that's not out yet, it needs to be wrapped so it tries for both yarn and intermediary names

Reply:
Fabric's reflection API half exists in the form of a Mapping resolver API. As such you can do something like this: https://github.com/Ladysnake/Dissolution/blob/fabric/src/main/java/ladysnake/requiem/common/util/reflection/ReflectionHelper.java#L45-L84

Reply:
Fixed with the latest commit; not sure if you like my double catch style though.